docs/docs/how-to/images-and-media/using-gatsby-plugin-image.md

```diff
@@ -10,27 +10,28 @@ The new Gatsby Image plugin is currently in beta, but you can try it out now and
 
 ## Getting started
 
-First you need to install the following packages:
+1. Install the following packages:
 
 ```shell
 npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-source-filesystem gatsby-transformer-sharp
 ```
 
-You then need to add the plugins to your `gatsby-config.js`:
+2. Add the plugins to your `gatsby-config.js`:
 
 ```js:title=gatsby-config.js
 module.exports = {
   plugins: [
     `gatsby-plugin-image`,
     `gatsby-plugin-sharp`,
-    `gatsby-source-filesystem`,
     `gatsby-transformer-sharp`,
   ],
 }
 ```
 
 If you already have some of these plugins installed, please check that they're updated to the latest version.
 
+Note that `gatsby-source-filesystem` is not included in this config. If you are sourcing from your local filesystem to use `GatsbyImage` please configure accordingly. Otherwise, downloading the dependency without configuration is sufficient.
+
```

Comment:
Is this going to be confusing?

Reply:
My instinct is that we have to say something? And having them configure it for an arbitrary folder they aren't sourcing seems unhelpful.

Reply:
cc: @meganesu Would love your take on this. Either way we need to change the existing config, so should figure out which option is the most helpful.